backend/lib/use-cases/orders/orders.ts

```diff
@@ -320,29 +333,41 @@ export class OrdersUseCase {
   }
 
   private async handleConsolidation(
-    context: ApplicationContext,
-    status: OrderStatus,
-    provisionalOrder: ConsolidationOrder,
-    includedCases: string[],
-    consolidationType?: ConsolidationType,
-    leadCase?: CaseSummary,
+    params: HandleConsolidationParams,
   ): Promise<ConsolidationOrder[]> {
+    const {
+      context,
+      status,
+      provisionalOrderId,
+      includedCases,
+      consolidationType,
+      leadCase,
+      reason,
+    } = params;
+
     if (!context.session.user.roles.includes(CamsRole.DataVerifier)) {
       throw new UnauthorizedError(MODULE_NAME);
     }
 
-    if (includedCases.length <= 1 && status === 'approved') {
+    // Included cases list does NOT include the lead case.
+    if (includedCases.length < 1 && status === 'approved') {
```

Comment:
I'd prefer to explicitly check that here, maybe something like:
```suggestion
    if (includedCases.length = 1 && status === 'approved' && includedCases[0].caseId === leadCase.caseId) {
```

This way the comment is not needed and our use case isn't depending on less-important code getting business logic right.